Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -105,11 +132,13 @@ static gboolean webKitMediaOpenCDMDecryptorHandleKeyResponse(WebKitMediaCommonEn
 
     GST_DEBUG_OBJECT(self, "handling session %s for event %u (ours %u)", session.get(), protectionEvent, priv->m_protectionEvent);
     if (priv->m_protectionEvent == protectionEvent) {
-        if (priv->m_session != session.get()) {
+        if (priv->m_openCdm == nullptr) {
```

Comment:
Why the change in logic here?

Reply:
From a CDM/DRM Session perspective, it should not change during the playback (can change from key but not from session), The logic know is assuming this and logs a debug message (in the else) if this happens. 
It is assumed that programatically the Session is set by now. Potentially we might add an if with a warning for that.

Reply:
I haven't checked the logic here, actually I am changing this in my branch.

Anyway, the style says it should be `if (priv->m_openCdm)`